src/graph_compute/numa_init.cpp

```diff
@@ -3,19 +3,6 @@
 
 // core_id = 0, 1, ... n-1, where n is the system's number of cores
 
-int bindThreadToCore(int _coreID) {
```

Comment:
Why delete this? Also, is the comment up on line 4 useful without this function?